README.md

```diff
@@ -14,8 +14,8 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://Zerg989.github.io/layout_landing-page/)
```

Comment:
Add this link to the PR description
<img width="943" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/64750958/6bb7169d-a5ae-4e8b-b29c-46802bade3aa">